versioned_docs/version-Release/alert-subscribe/easy-subscribe/index.md

```diff
@@ -0,0 +1,110 @@
+
+# Easy Subscribe
```

Comment:
FIX: or manage notification preferences. 

FIX:  Users can subscribe using their email address and have the option to receive notifications via a Telegram bot.